.github/workflows/ubuntu-system.yml

```diff
@@ -63,20 +69,42 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: 3.x
-          
+
     - name: Install dependencies
       run: |
           python -m pip install --upgrade pip
           pip3 install -r src/tests/examples/requirements.txt
-           
+
     - name: Init submodule
       run: |
            git submodule update --init --recursive src
 
+    - name: Download XPRESS if needed
+      if: ${{ matrix.ortools == 'ON' }}
+      run: |
+        git clone -b master https://${{ secrets.XPRS_USERNAME }}:${{ secrets.XPRS_TOKEN }}@github.com/rte-france/xpress-mp ${{ env.XPRESSDIR }}
+
+    - name: Download OR-Tools if needed
+      if: ${{ matrix.ortools == 'ON' }}
+      id: ortools
+      run: |
+          if [ "${{ matrix.ortools }}" == "ON" ]; then
+            mkdir -p ${{ env.ORTOOLSDIR }} && cd ${{ env.ORTOOLSDIR }}
+            wget -q -O ortools.zip https://github.com/rte-france/or-tools/releases/download/v9.2-rte1.3/ortools_cxx_ubuntu-20.04_shared_xprs.zip
```

Comment:
Move this to an env var
* `matrix.ortools == 'false'` => https://github.com/rte-france/or-tools/releases/download/v9.2-rte1.3/ortools_cxx_ubuntu-20.04_static_sirius.zip
* `matrix.ortools == 'true'` => https://github.com/rte-france/or-tools/releases/download/v9.2-rte1.3/ortools_cxx_ubuntu-20.04_static_sirius_xprs.zip

Remove specific build flags (l. 97 and 100), always pass `-DUSE_XPRESS=${{ matrix.ortools }} -DXPRESS_ROOT=${XPRESSDIR}` (l. 106). If XPRESS is not used, XPRESS_ROOT will be unused but it doesn't matter.